app/views/booths/_form.html.haml

```diff
@@ -10,6 +10,14 @@
         = f.input :submitter_relationship, input_html: { rows: 5, data: { provide: 'markdown-editable' } }, required: true,
           label: 'Submitter\'s relation',
           hint: 'e.g. employee, comunity manager, etc'
+
+        = f.input :booth_type_id, as: :select,
+          collection: conference.program.booth_types.map {|type| ["#{type.title}", type.id]},
+          include_blank: false, label: 'Type', input_html: { class: 'select-help-toggle' } if conference.program.booth_types.any?
```

Comment:
Perhaps it doesn't always make sense to enforce the selection of the type. I'd suggest we do include the blank option with a relevant message (Please select, or sth similar to our other forms)

Reply:
@differentreality sure :+1: